### ISSUE TITLE
Add previous Safepaths papers 

### ISSUE THREAD

**Issue by tremblerz** [Path-Check/vaccine-diary#4]
[Link to our list of papers](https://github.com/privatekit/privacyDocuments)

**Comment by Sheshank-s** [comment]
Not all of these papers are directly relevant to vaccine-diary though, should we still add them? I think it might be better to just link to arXiv for any paper that isn't directly related.

**Comment by tremblerz** [comment]
Good point! Let's add a README file with information about and link to other papers. That way we don't have to update two repositories separately.
